templates/tweets/edit.html

```diff
@@ -3,7 +3,7 @@
 <div>
   <a href="{% url 'home' %}" style="float: right">Home</a>
   <h4>Edit Tweet</h4>
-  <form method="post" action="{% url 'tweets:updateTweet' tweet.id %}">
+  <form method="post" action="{% url 'tweets:DetailTweet' tweet.id %}">
```

Comment:
patch ?

Reply:
yes this is not done as of now. I need to confirm the edit script then will move on to this. athough till then it is working by default post function in views.